app/common/data/types.py

```diff
@@ -68,6 +68,15 @@ class SubmissionStatusEnum(enum.StrEnum):
     COMPLETED = "Completed"
 
 
+class TasklistTaskStatusEnum(enum.StrEnum):
+    NOT_STARTED = SubmissionStatusEnum.NOT_STARTED.value
+    IN_PROGRESS = SubmissionStatusEnum.IN_PROGRESS.value
+    COMPLETED = SubmissionStatusEnum.COMPLETED.value
+    assert len(SubmissionStatusEnum) == 3
```

Comment:
is this hanging over from something or left in to intentionally make you change this when adding more? maybe a comment if so